### PR TITLE
Lend integration

### PR DESCRIPTION
**Pull request by matzapata**
## Overview

Jira ticket: [SAF-8](https://wakeuplabs.atlassian.net/browse/SAF-8)

Connect supply forms with contracts.

## Extra comments

On hold until we remove polygon and testing in bob is possible.

### CODE REVIEW

**Comment by matzapata** [comment]
Demo

https://github.com/user-attachments/assets/53fb9d16-f9bc-40c7-b119-f5b9685709de

**Review by njaume — APPROVED** [review]
Great!

File: apps/frontend/src/utils/aave/AaveUserReservesSummary.ts [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  apyType: ApyType;
+  borrowed: Decimal;
+  borrowedUSD: Decimal;
+};
+
+export class AaveUserReservesSummary {
+  public netWorth: Decimal;
+  public netApy: Decimal;
+  public healthFactor: Decimal;
+  public supplyBalance: Decimal; // OK
```
Comment (juandahl) [inline apps/frontend/src/utils/aave/AaveUserReservesSummary.ts]:
```suggestion
  public supplyBalance: Decimal;
```

**Review by juandahl — APPROVED** [review]
LGTM

**Pull request merged**